### DISCUSSION TITLE
Custom Field for FormBuilder Plugin

### DISCUSSION THREAD

**Discussion by DesignyourCode** [payloadcms/payload#10104] [General]
Hi. I am trying to add my own custom field to the form builder. I have started with the website template.

But I am struggling to understand how I add any new fields to it.

What I would like, is to have a dropdown select field similar to the way the country field works, but to have the options populated using the list of docs from a collection.

However, the documentation for the plugin doesn't seem to provide any examples of adding custom fields, and the syntax doesn't appear to be consistent with the way the rest of the collection/block configs work. Can anyone provide me with some help here?

Thanks

**Comment by dkirchhof** [comment]
Duplicate? https://github.com/payloadcms/payload/discussions/7161
